src/vstest.console/Resources/Resources.resx

```diff
@@ -236,8 +236,9 @@
       Visual Studio Test Results File (TRX) use /logger:trx [;LogFileName=&lt;Defaults to unique file name&gt;]
       Creates file in TestResults directory with given LogFileName.
 
-      Change the verbosity level for console logger. Allowed values for verbosity: quiet, minimal and normal.
-      Example: /logger:console;verbosity=&lt;Defaults to "minimal"&gt;
+      Change the verbosity level and add the verbosity level as prefix in log messages for console logger. Allowed values for verbosity: quiet, minimal and normal.
```

Comment:
This string does not make sense for me.
It is a bit confusing, how can I enable/disable adding prefix is not clear

Reply:
Suggestions are welcome.

Reply:
@pvlakshm